samila/functions.py

```diff
@@ -169,6 +169,22 @@ def filter_color(color, bgcolor):
     return color, bgcolor
 
 
+def get_cmap(name=DEFAULT_CMAP_NAME, lut=256):
+    """
+    Get colormap.
+
+    :param name: colormap name
+    :type name: str
+    :param lut: look up table for colormap
+    :type lut: int
+    :return: desired colormap
+    """
+    try:
+        return matplotlib.colormaps.get_cmap(name)
+    except BaseException:
```

Comment:
Is there any way to test this line of code? (something like `del matplotlib.colormaps.get_camp`?!)
If the answer is no, please add `# pragma: no cover` to this exception.

Reply:
> Take a look at my comment about tests and coverage. Also, it seems `notebook check` has a problem, solve the problem temporarily

I ran it and it was totally OK for my system. I will check the test and solve the problem if needed.

Reply:
> del matplotlib.colormaps.get_camp

Actually, it seems it's not the best practice since we need the current version of the matplotlib to support both of the methods to pass the test. The best option we have is to use the `# pragma: no cover`.